otter/http.py

```diff
@@ -257,6 +257,11 @@ def perform_tenant_scope(
     to be partially applied, and the result is a performer that can be put into
     a dispatcher.
     """
+    if _concretize is None:
+        # The reason this isn't set as the default kwarg value above is because
+        # of a mock in test_metrics.py.
```

Comment:
oh no, it wasn't a pain for me -- I added the mock myself. I'm sure it'll be a pain for the next person that tries to change some mock though ;)